src/main.js

```diff
@@ -6,8 +6,12 @@ import store from "./store";
 import upperFirst from "lodash/upperFirst";
 import camelCase from "lodash/camelCase";
 
+import $ from "jquery";
+window.$ = window.jQuery = $;
+
 import "bootstrap";
 import "bootstrap/dist/css/bootstrap.min.css";
+import "bootstrap/dist/js/bootstrap.bundle.min.js";
```

Comment:
Soweit mir bekannt, nutzt Bootstrap in der Version 4.6 sehr viel jquery. Erst in der Version 5.x wurde das komplett entfernt. Bitte das gewünschte Verhalten ohne jquery und das Bootstrap JS umsetzen.